01 - Community/Obsidian Roundup/2022-11-12 Stable Diffusion, Clean Dashboards and 2022 Workflows.md

```diff
@@ -0,0 +1,99 @@
+---
+link: https://www.obsidianroundup.org/2022-11-12/
+author: Curtis McHale
+published: 2022-11-12T13:30:50
+publish: true
+---
+
+# 2022-11-12: Stable Diffusion, Clean Dashboards and 2022 Workflows
+A new social media instance for personal knowledge management & a theme to bring back old-style Obsidian.
+
+💎Eleanor got Lasik on Thursday so she's trying to take it easy this weekend. This issue was guest edited by [Curtis McHale](https://curtismchale.ca).## In The Community
```

Comment:
I will log an issue for the `## In The Community` not being at the start of a line.

Reply:
Logged in https://github.com/obsidian-community/obsidian-hub/issues/535